### ISSUE TITLE
Front: When loading additional row in a table (issues, security, suggestions...) width become unbound

### ISSUE THREAD

**Issue by Driky** [exakat/exakat#61]
While testing exakat I found the following bug:

When you scroll down on a page with a long table where many rows are loaded scroll and it load those extra rows, the width of the table becomes unbounded. Mine become more than 8000 pixels on firefox nightly 62.0a1 and on chrome Version 62.0.3202.62 (Official Build) (64-bit).

![screenshot from 2018-06-08 14 22 38](https://user-images.githubusercontent.com/1233974/41174306-c51a5cce-6b27-11e8-938b-571db6c5da84.png)

**Comment by exakat** [comment]
Yes this happens in particular if the table is long, and the code snippets are large.  There should be some limit in the size of those snippets.
